readme.md

```diff
@@ -152,14 +157,12 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
-# Long Term Support
+# Long-Term Support (LTS)
 
 The project offers a Long Term Support (LTS) version to allow users that do not need the latest features to be shielded from unwanted changes.
 
-As of August 2020, the .NET SDK is shifting to a revised Long-Term Support strategy. The primary motivations for this change are to extend the support period and decrease the churn on LTS releases, while still maintaining a strategy that offers customers choice between new features and stability. 
```

Comment:
This text was provided by Elena to highlight our LTS strategy, and the motivation behind supporting it for 3 years. I feel we should retain this for one LTS release cycle.

Reply:
This change was unintnentional. I will restore it.